modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/java/JavaModelLoaderDelegate.java

```diff
@@ -168,6 +177,34 @@ public ExtensionDeclarer declare(ExtensionLoadingContext context) {
     return declarer;
   }
 
+  protected void parseExternalLibs(WithAnnotations withAnnotations, DeclaresExternalLibraries declarer) {
+    Optional<ExternalLibs> externalLibs = withAnnotations.getAnnotation(ExternalLibs.class);
```

Comment:
you can use `MuleExtensionAnnotationParser#parseRepeatableAnnotation` like we do for subtypes: 
`List<SubTypeMapping> typeMappings = parseRepeatableAnnotation(type, SubTypeMapping.class, c -> ((SubTypesMapping) c).value());`